charts/homarr/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: homarr
 description: A Helm chart to deploy homarr for Kubernetes
 home: https://homarr-labs.github.io/charts/charts/homarr/
 type: application
-version: 0.2.0
+version: 0.2.1
 # renovate datasource=docker depName=ghcr.io/homarr-labs/homarr
 appVersion: "0.0.1"
```

Comment:
I think this should be the beta version

Reply:
It is the chart version not the homarr version